src/components/common/date-input/DateInput.tsx

```diff
@@ -17,32 +43,52 @@ type Props = {
 };
 
 const defaultProps = {
+  dense: false,
   placeholder: "",
   testId: "",
 };
 
-const DateInput = ({ id, onChange, placeholder, testId, value }: Props) => (
-  <KeyboardDatePicker
-    autoOk
-    disableFuture
-    disableToolbar
-    format="MM/DD/yyyy"
-    fullWidth
-    id={id}
-    inputProps={{ "aria-label": testId, "data-testid": TestId.Input }}
-    inputVariant="outlined"
-    KeyboardButtonProps={{
-      "aria-label": "change date",
-      ...{ "data-testid": TestId.KeyboardButton },
-    }}
-    onChange={(date) =>
-      onChange(date ? moment(date, "MM/DD/yyyy").toDate() : null)
-    }
-    placeholder={placeholder}
-    value={value}
-    variant="dialog"
-  />
-);
+const DateInput = ({
```

Comment:
this change looks big because the content was indented - main change here was adding `className` properties for styling